reference_validator/YAML_Enums.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+#coding=utf-8
+
+# File: YAML_Enums.py
+# Brief: Additional classes defining enumeration types for HOT reference validator
+# Classes: YAML_colours, YAML_tree_info, YAML_Types
+# Author: Katerina Pilatova (kpilatov)
+# Date: 2016
+
+class YAML_colours:
```

Comment:
I spell it `colour` too but iirc OpenStack (which we hope this will be a part of at some point) generally prefers american spelling.